tests/unit_tests/data_steward/cdr_cleaner/cleaning_rules/unit_normalization_test.py

```diff
@@ -0,0 +1,110 @@
+# Python imports
+import unittest
+
+from cdr_cleaner.cleaning_rules.unit_normalization import UnitNormalization, \
+    UNIT_NORMALIZATION_QUERY, SANDBOX_UNITS_QUERY, UNIT_MAPPING_TABLE, MEASUREMENT
+# Project imports
+from constants.bq_utils import WRITE_TRUNCATE
+from constants.cdr_cleaner import clean_cdr as clean_consts
+
+QUERY = 'query'
+
+
+class UnitNormalizationTest(unittest.TestCase):
+
+    @classmethod
+    def setUpClass(cls):
+        print('**************************************************************')
+        print(cls.__name__)
+        print('**************************************************************')
+
+    def setUp(self):
+        self.project_id = 'foo_project'
+        self.dataset_id = 'foo_dataset'
+        self.sandbox_id = 'foo_sandbox'
+        self.client = None
+
+        self.query_class = UnitNormalization(self.project_id, self.dataset_id,
+                                             self.sandbox_id)
+
+        self.assertEqual(self.query_class.get_project_id(), self.project_id)
+        self.assertEqual(self.query_class.get_dataset_id(), self.dataset_id)
+        self.assertEqual(self.query_class.get_sandbox_dataset_id(),
+                         self.sandbox_id)
+
+    def test_setup_rule(self):
+        # Test
+        def run_setup(client):
+            try:
+                self.query_class.setup_rule(client)
+            except RuntimeError:
+                return
+
+        self.assertRaises(RuntimeError, run_setup(self.client))
```

Comment:
Sorry, not sure the idea behind `run_setup(client)` above- here's my suggestion.
```suggestion
        with self.assertRaises(RuntimeError) as c:
             self.query_class.setup_rule(None)
        self.assertEqual(str(c.exception), 'Specify BigQuery client object')
```